File: JS/Conditional Statements/SumSeconds.js
function sumSeconds(input){
    let first = parseInt(input[0]);
    let second = parseInt(input[1]);
    let third = parseInt(input[2]);
    let totalSeconds = first + second + third;
    let minutes = Math.floor(totalSeconds / 60);
    let seconds = totalSeconds % 60;

    if (seconds < 10){
        seconds = `0${seconds}`;
    }
    console.log(`${minutes}:${seconds}`);
};

// sumSeconds(["35", "45", "44"])

File: Python Basics/exercises/sum_seconds.py
import math

time_first = int(input())
time_second = int(input())
time_third = int(input())

summary_time = time_third + time_second + time_first
minutes = summary_time / 60
seconds = summary_time % 60
minutes = math.floor(minutes)

if seconds < 10:
    print(f"{minutes}:0{seconds}")
else:
    print(f"{minutes}:{seconds}")